applications/FluidDynamicsApplication/tests/test_apply_mach_dependent_boundary_conditions.py

```diff
@@ -0,0 +1,514 @@
+import KratosMultiphysics
```

Comment:
Note that all the tests in the `FluidDynamicsApplication` have the keyword `test` at the end of the filename or class name. This is intentionally done to ease browsing the code. Please do it accordingly.